vcluster/deploy/security/fips.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: FIPS 140-2 Enablement
+sidebar_label: FIPS 140-2
+sidebar_position: 7
+sidebar_class_name: pro
+---
+
+import ProAdmonition from '@site/vcluster/_partials/admonitions/pro-admonition.mdx'
+
+The National Institute of Standards and Technology (NIST) develops Federal
+Information Processing Standards (FIPS) to ensure the security and
+interoperability of computer systems used by the U.S. government.
+
+FIPS 140-2 is a U.S. Federal Government security standard used to approve
```

Comment:
Why are we referencing FIPS 140-2 and not FIPS 140-3, which is now the newer requirement?